pages/documentation/grid/model/index.md

```diff
@@ -615,14 +693,19 @@ In between the defined points of the curve, the reactive power limits are comput
 Some equipment have operational limits regarding the current value, corresponding to the equipment's physical limitations (related to heating).
 The current limits may be set in IIDM for [lines](#line),
 [dangling lines](#dangling-line), [two windings transformers](#two-windings-transformer) and [three windings transformers](#three-windings-transformer).
-Current limits are defined by at most one permanent limit and/or any number of temporary limits.
+Current limits are defined by at most one permanent limit and any number of temporary limits (zero or more).
 The permanent limit sets the current value (in `A`) under which the equipment can safely
 be operated for any duration.
 The temporary limits can be used to define higher current limitations corresponding
 to specific operational durations.
 A temporary limit thus has an **acceptable duration**.
 The component on which the current limits are applied can safely remain
-between the preceding limit (it could be another temporary limit or a permanent limit) and this limit for a duration up to the acceptable duration.
+between the preceding limit (it could be another temporary limit or a permanent limit) and this limit for a duration up to the acceptable duration. Please look at this scheme to fully understand the modelling:
+
+![Current limits model](img/index/currentLimits.svg){: width="50%" .center-image}
+
+Note that, following this modelling, the last temporary limit (the higher one in value) should be infinite with an acceptable duration 
+different from zero.
```

Comment:
Shouldn't we allow to set the acceptable duration to zero for the tripping current?